scalafix-core/shared/src/main/scala/scalafix/patch/Patch.scala

```diff
@@ -48,12 +48,14 @@ sealed abstract class Patch {
   def ++(other: Iterable[Patch]): Patch = other.foldLeft(this)(_ + _)
   def isEmpty: Boolean = this == EmptyPatch
   def nonEmpty: Boolean = !isEmpty
+  def atomic: Patch = AtomicPatch(this)
```

Comment:
Add docstring. The other methods have no docstring since I think they're quite self explanatory but atomic is not as obvious

Reply:
I will write user documentation with examples and limitation.

Reply:
We should still have a docstring despite docs on the website, a single sentence like

> Skip this entire patch if a part of it is disabled with `// scalafix:off`

will do.